projects/hdfgroup.org/HDF5/package.yml

```diff
@@ -0,0 +1,72 @@
+distributable:
+  url: https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-{{version.marketing}}/hdf5-{{version}}/src/hdf5-{{version}}-2.tar.bz2
+  strip-components: 2
+
+versions:
+  - 1.14.1
```

Comment:
I'm not sure about the versions. On the page https://support.hdfgroup.org/ftp/HDF5/releases/, there is no list of all versions, only folders with {{version.marketing}}. And the latest version has an additional `-2` in the file name (`1.14.1-2.tar.bz2`), while the others don't (`1.14.0.tar.bz2`, `1.13.3.tar.bz2`).

There is only one version available on GitHub (`hdf5-1_14_1-2`).

Reply:
Yeah, they don't make it easy. Might be the best we can do for now.